### ISSUE TITLE
Language names in "First Launch" are not translated

### ISSUE THREAD

**Issue by dyskette** [Bionus/imgbrd-grabber#740]
As you can see in this screenshot, the names are from the `.qm` translation files.

![captura de pantalla de 2017-01-10 13-48-07](https://cloud.githubusercontent.com/assets/6687927/21822695/db0594ec-d73d-11e6-9374-b6b0189e107f.png)

However, they show correctly in the [Options window.](https://cloud.githubusercontent.com/assets/6687927/21822696/db0685a0-d73d-11e6-926b-0bdde6f45dd2.png)

- **What steps will reproduce the problem?**

Open "First Launch" window.

- **What is the expected behavior? What do you get instead?**

To see the language's own translations. Instead they are the TranslationFiles.qm without the extension.

- **What version of the program are you using? On what operating system?**

Grabber 5.0.1 on Fedora 25 (Linux).

**Comment by Bionus** [comment]
Indeed, it seems that the language labels from the txt file are only applied in the options window.

BTW, why do you have the `YourLanguage` language popping up? Did you `lrelease` the project or was it already there?

**Comment by dyskette** [comment]
Yeah, used `lrelease`. I made a package for Fedora to install it and didn't notice. I'll remove it later.

BTW, there is another bug concerning languages and installation on Linux: Grabber doesn't find translations files in the system. 

Let's say I made the `PREFIX=/usr`, then install using `make install`. Because I cannot put translation files in `/usr/bin` (only executable unique named binaries there) and I cannot modify the `$HOME` folder of the user to place translation files on installation (and would bring another set of issues), there is nowhere else to put them. I would suggest that Grabber tries to find them in `PREFIX/share/Grabber/languages`
